src/domains/Find/detail/ChangePwPage.jsx

```
@@ -19,57 +21,78 @@ const ChangePwPage = () => {
       if (!isValid) {
-        // requireResetToken 내에서 이미 리다이렉트 처리
+        console.log('비밀번호 확인이 필요합니다.2')
+        ToastService.info('비밀번호 확인이 필요합니다.')
       }
     }
     checkToken()
   }, [requireResetToken, navigate])
 
   const onSubmit = async (data) => {
-    const { passwordCheck, ...filteredData } = data // passwordCheck 필터링
+    const { passwordCheck, ...filteredData } = data
```

ESLint

[error] 'passwordCheck' is assigned a value but never used.

(no-unused-vars)